datarobot_provider/example_dags/model_compute_shap_dag.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from datetime import datetime
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.model_insights import ComputeShapOperator
+from datarobot_provider.sensors.model_insights import DataRobotJobSensor
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2023, 1, 1),
+    tags=['example', 'insights'],
+)
+def compute_model_shap(project_id=None, model_id=None):
+    if not project_id:
+        raise ValueError("Invalid or missing `project_id` value")
+    if not model_id:
+        raise ValueError("Invalid or missing `model_id` value")
+
+    compute_shap_op = ComputeShapOperator(
+        task_id="compute_feature_impact",
+        project_id=project_id,
+        model_id=model_id,
+    )
+
+    shap_complete_sensor = DataRobotJobSensor(
+        task_id="feature_impact_complete",
+        project_id=project_id,
+        job_id=compute_shap_op.output,
```

Comment:
Maybe I haven't been following along close enough, but I see that the operator returns the ID field but so is `.output` just a built-in thing to Airflow operators?

Reply:
yes its a built-in thing in Airflow, because we transmitting output of this operator using xcom under the hood